swagger/v1/component_schemas/NPQParticipantProfileAttributes.yml

```diff
@@ -0,0 +1,27 @@
+description: "The data attributes associated with an NPQ application"
+type: object
+required:
+  - participant_id
+  - training_status
+properties:
+  id:
+    type: string
+    example: 7a8fef46-3c43-42c0-b3d5-1ba5904ba562
+    format: uuid
+  participant_id:
+    description: "The unique identifier of this NPQ participant"
+    type: string
+    example: 7a8fef46-3c43-42c0-b3d5-1ba5904ba562
+    format: uuid
+  course_identifier:
+    description: "The NPQ course identifier"
+    type: string
+    example: moved-school
```

Comment:
This is not in the attributes for participant profile - it will be in the request shape though.

Reply:
Oh, sorry, I just didn't see this in the serializer.